src/main/java/seedu/address/logic/commands/UnpinCommand.java

```diff
@@ -46,12 +46,16 @@ public CommandResult execute(Model model) throws CommandException {
             peopleToUnpin.add(personToUnpin);
         }
 
+        assert peopleToUnpin.size() == targetIndices.size();
```

Comment:
Good that you are using defensive programming techniques such as assert in our code